discovery-provider/src/queries/get_notifications.py

```diff
@@ -82,8 +84,10 @@
    select n.type, n.group_id
    from
        notification n
+   join users u on
+       u.user_id = :user_id
```

Comment:
this join will duplicate rows for each user row. we really only need 1 user row and that should come from a CTE